src/core/frontend/manager/UIManager.cpp

```diff
@@ -31,7 +31,7 @@ namespace YimMenu
 		}
 		ImGui::EndChild();
 		
-		ImGui::Text("HorseMenu");
+		ImGui::Text("Game Ver. 1.32"); // Game version
```

Comment:
You could do something like 
`HorseMenu - Game Ver. 1.32`

Reply:
True, but it seems unnecessary to add horsemenu at the bottom text with the new change.

Reply:
yeah, either or it. It is a small detail so it doesn't matter much. Are there any natives to get the game version? those could be used if there are

Reply:
Not sure but i will check tonight and see.  Had some PC issues from a windows update when i woke up today so reinstalled windows before i left for work.